### PR TITLE
[README] Update README files

### PR DESCRIPTION
**Pull request by markavitale**
Update the readme files in this repo to better reflect the usage and contribution patterns.

### CODE REVIEW

**Comment by markavitale** [comment]
Note I recommend using the "Rich Diff" option to see how the README will display on GitHub.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,66 @@
...
+#### Using CocoaPods
+
+To get set up with CocoaPods visit their [getting started guide](https://guides.cocoapods.org/using/getting-started.html).
+
+To integrate FluentUI into your Xcode project using CocoaPods, specify it in your Podfile:
+```ruby
+pod 'MicrosoftFluentUI', '~> X.X.X'
+```
+
+then follow the Carthage [integration steps](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) to add the `FluentUI.framework` into your Xcode project
```
Comment (harrieshin) [inline README.md:36]: i think we should remove this line. better to just run "pod install" and follow cocoapod guidelines ....

File: ios/README.md [hunk 2]
```diff
@@ -23,94 +20,33 @@ FluentUI for iOS provides [colors](FluentUI/Core/Colors.swift) and [typography](
...
-- MSNotificationView
-- MSPersonaListView
-- MSPillButtonBar
-- MSPopupMenuController
-- MSSegmentedControl
-- MSShimmerLinesView
-- MSTabBarView
-- MSTableViewCell
-- MSTooltip
+- ActivityIndicatorView
```
Comment (harrieshin) [inline ios/README.md:23]: need a rebase? i thought i changed these yesterday...

File: README.md [hunk 3]
```diff
@@ -1,14 +1,66 @@
...
+import FluentUI
+```
+For Objective-C
+```objective-c
+#import <FluentUI/FluentUI.h>
+```
+
+## Contributing
+
+Post bug reports, feature requests, and questions in [Issues](https://github.com/microsoft/fluentui-apple/issues).
```
Comment (harrieshin) [inline README.md:63]: maybe a todo: should we consider adding guidelines about when creating a new control make sure it is added for both framework and static library target, and there should be an example in the demo/test app? maybe we should have a wiki page of todo checklist: accessibility/light/darkmode etc

File: README.md (hunk 1 above)
Comment (markavitale) [reply README.md:36]: Whoops this belongs up with the Carthage section, not in the CocoaPods section. I'll move it.

File: README.md (hunk 3 above)
Comment (markavitale) [reply README.md:63]: I wonder if that would make sense in a PR template? Like under a verification section, have a checklist of whether your change impacts Ax, dark mode, RTL, localization, etc and how you tested it?

Comment (markavitale) [reply README.md:63]: We can also have separate bug templates for bug reports and feature requests if we want. that information tracked at the issue level.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,66 @@
...
+
+- Download the latest changes from the [FluentUI for Apple](https://github.com/microsoft/fluentui-apple) repository.
+- Move the `fluentui-apple` folder into your project folder.
+- Move the relevant `FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
+- In Xcode select your project -> your target -> General -> Embedded Binaries -> add `FluentUI.framework`.
+
+#### Swift Package Manager
+As of this writing, the version of Swift Package Manager shipped with the latest Xcode does not support packages that require resource bundles. As Fluent UI Apple does  require resource bundles, we do not currently support Swift Package Manager.
+
+### Import and use the library
```
Comment (anandrajeswaran) [inline README.md:48]: Is the word "library" correct here? At the least it's confusing to me. But then my related question is that these instructions are specific to using it as a framework and not a static library?

File: macos/README.md [hunk 5]
```diff
@@ -1,14 +1,30 @@
...
+- [Controls](#controls)
+- [Test app](#test-app)
+
+## Controls
+
+FluentUI for macOS includes an expanding library of controls written in Swift and supporting Objective-C. These controls implement the Fluent Design language and provide consistency across Microsoft experiences.
+
+Some of the controls available include:
+- AvatarView
+- DatePicker
```
Comment (anandrajeswaran) [inline macos/README.md:18]: When we have a more fully featured control set we'll probably want to remove it, right? Should we just do that now?

File: macos/README.md [hunk 6]
```diff
@@ -1,14 +1,30 @@
...
+
+A full list of currently supported controls can be found here: [FluentUI](FluentUI).
+
+## Test app
+
+Included in this repository is a demo of currently implemented controls. A full list of implemented controls available in the test app can be found here:  [TestApp](FluentUI.Demo/FluentUI.Demo/Demos).
```
Comment (anandrajeswaran) [inline macos/README.md:24]: Should we call this"Demo" instead of TestApp/Test app?

Comment (anandrajeswaran) [reply macos/README.md:24]: Just for consistency

File: README.md [hunk 7]
```diff
@@ -1,14 +1,66 @@
-# FluentUI Apple
-Fluent UI Apple is a repository containing native UIKit and AppKit controls aligned with Microsoft's Fluent UI design language. 
+# Fluent UI Apple
+Fluent UI Apple is a library containing native UIKit and AppKit controls aligned with [Microsoft's Fluent UI design system](https://www.microsoft.com/design/fluent/#/).
```
[the remark this answers is not in the document]
Comment (markavitale) [reply README.md]: Removed here, and reworded elsewhere. When specifically referencing build output, I used the word "framework"

File: README.md (hunk 4 above)
Comment (markavitale) [reply README.md]: Reworded.

**Pull request merged**